test/cli_spec.js

```diff
@@ -229,7 +240,7 @@ describe("CLI Tests", function () {
       "--custom-header",
       "k1: v1",
       "--custom-header",
-      " k2 : v2 v2 ",
+      " k2 : host:123 ",
```

Comment:
Maybe adding a test with an invalid header (eg `--custom-header badheader`) would be worthwile ? I think the current error logging in `collectHeadersIntoObject` crashes.